custom_metrics/iframes.js

```diff
@@ -0,0 +1,181 @@
+//[iframes]
+// Uncomment the previous line for testing on webpagetest.org
+
+// Sanitize the `attributes` property.
+function getNodeAttributes(node) {
```

Comment:
Given the duplication of code in [almanac.js](https://github.com/HTTPArchive/legacy.httparchive.org/blob/master/custom_metrics/almanac.js) would you be open to adding this custom metric there as a new `resource_hints` property of the response object? You could also include #202 in there as well.

I'd prefer not to maintain core functionality like these functions across separate files.

Reply:
Updated. Also updated description with new WPT link and JSON.